Community Rules Rework

Ephemeralis opened the pull request:
Give this a good ogle and lookover. It's been through 3 passes in the GMT and a month of revision from the community proper, so it should be okay, but still.

This will require new translations for all other languages since it is for the most part, a full wording rewrite.

TPGPL commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
 osu! Rules
```
rename title to Rules

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
+6.  **Don't post advertisements, referral links or any other annoying or intrusive content.** Things like affiliate and referral links are fun for the person posting them and an annoyance for everybody else, so don't do it.  Autoloading or autoplaying content/embeds falls within this rule as well.
+7.  **Linking to applications or utilities of uncertain intent is not allowed.** Put simply, if there is any doubt as to the legality of said utilities, they are not welcome to be posted.
+
+[Beatmap Submission Rules](BSS "wikilink")
```
remove link from heading.... if you really want it, please add:
```
_See also: [Beatmap Submission Rules](/wiki/BSS)_
```

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
 ---------------------------------
 
-These rules apply strictly to the unique nature of the [BSS](/wiki/Glossary) ([Beatmap](/wiki/Glossary) Submission System) threads. These threads are created via the osu! client when a beatmap is submitted through the [editor](/wiki/Beatmap_Editor).
+These rules apply strictly to beatmaps submitted via the in-game [Beatmap Submission System](BSS "wikilink"). These threads/discussions are created via the osu! client when a beatmap is submitted through the [editor](editor "wikilink").
```
Fix these links:
- `[Beatmap Submission System](BSS "wikilink")` ↙️
  - `[Beatmap Submission System](/wiki/Beatmap_Submission_System)`
- `[editor](editor "wikilink")` ↙️
  - `[editor](/wiki/editor)`

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
-These rules apply strictly to the unique nature of the [BSS](/wiki/Glossary) ([Beatmap](/wiki/Glossary) Submission System) threads. These threads are created via the osu! client when a beatmap is submitted through the [editor](/wiki/Beatmap_Editor).
+These rules apply strictly to beatmaps submitted via the in-game [Beatmap Submission System](BSS "wikilink"). These threads/discussions are created via the osu! client when a beatmap is submitted through the [editor](editor "wikilink").
+
+A set of more detailed rules for beatmap construction exists, known as the [Ranking Criteria](Ranking_Criteria "wikilink").
```
Fix this link:
- `[Ranking Criteria](Ranking_Criteria "wikilink")` ↙️
  - `[Ranking Criteria](/wiki/Ranking_Criteria)`

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
-1.  Do not create threads in the [Pending Submissions](/wiki/Beatmaps) and [WIP](/wiki/Glossary) boards. Use the appropriate subforum when creating a thread. (This obviously excludes BSS threads.)
-2.  Do not get off topic in BSS threads. BSS threads are about the parent beatmap. If you have an issue with something not directly pertaining to the beatmap, post it in the appropriate forum.
-3.  Do not steal others' beatmaps, presenting them as if they're mapped by you. Do not also upload or use other people's work without their explicit permission (including, but not limited to skins and guest difficulties).
+1. **Make sure you have permission to use any content involved in your beatmap.** This includes songs, videos, hitsounds, graphics, and any other content that isn't your own creation. If you are unsure of where to find content that is free to use, we have a growing library of tracks available at the [Featured Artist](Featured_Artist "wikilink") listing, all 100% licensed for use in osu!.
```
The link `[Featured Artist](Featured_Artist "wikilink")` does **not** exist 😕 
...
If you still want to link to it anyways, fix to:
- `[Featured Artist](Featured_Artist "wikilink")` ↙️
  - `[Featured Artist](/wiki/Featured_Artist)`

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
+* The removal of your ability to chat in-game for a period of time (known as a silence)
+* The removal of any infringing content from your userpage, profile or uploaded beatmaps
+* The loss of the privilege to edit or alter your userpage or profile details
+* The loss of the privilege to play or qualify in official osu! World Cup tournaments
```
nitpicking, but `official osu! World Cup tournaments` sounds like it only affects the osu!(standard) ones :/

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
+Silences and other infringements **automatically** increase in duration near-exponentially with each infringement. Moderators do not control the length of time you are silenced for an offense - repeat offenders will find themselves locked out of the game for increasingly longer periods of time.
 
-Violating any of these rules results in the offender being hit with either a silence or a restriction.
+Certain offenses carry a minimum duration (usually 48hrs or more) if they are particularly severe, though they are almost exclusively reserved for severe infractions, such as sharing graphic content or unrepentant harassment of other users.
```
write out `hrs`?

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
 
-Violating any of these rules results in the offender being hit with either a silence or a restriction.
+Certain offenses carry a minimum duration (usually 48hrs or more) if they are particularly severe, though they are almost exclusively reserved for severe infractions, such as sharing graphic content or unrepentant harassment of other users.
 
```
horizontal bar to start the "conclusion" of "if I break the rules?" ❓

Ephemeralis replied:
explain what you mean?

Ephemeralis commented:
I've changed the FA link to point directly to the website, fixed the links and so on. Unsure of what you mean with "horizontal bar to start the conclusion" though.

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
+Certain offenses carry a minimum duration (usually 48 hours or more) if they are particularly severe, though they are almost exclusively reserved for severe infractions, such as sharing graphic content or unrepentant harassment of other users.
 
-Silenced users cannot chat, post on the forums, update their beatmaps, or play multiplayer games.
+If after several infringements you continue to break the rules, your account may be permanently and irrevocably restricted. 
```
remove trailing whitespace (there is a space after the full stop, remove it)

MegaApplePi commented:
@Ephemeralis Please disregard about the "add horizontal bar", I misread the parts of the `What happens if I break the rules?` section.

Just to point it out, a horizontal bar is the HTML `<hr/>` tag... but in markdown, use three hyphens `---` (making sure that there is an empty line above it)

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
-BSS Forum Rules
+1.  **Post only meaningful, interesting or otherwise engaging content/topics.** Forums are about discussions foremost, so make sure to put the effort in to make it worth everyone's time.
+2.  **Keep your posts and replies suitable for people of all-ages.** NSFW content isn't appropriate for an all-ages community, though there's nothing stopping you from discussing these sorts of things in private messages.
+3.  **Be productive with your criticism without resorting to personal attacks.** Criticism is a wonderful thing when done properly, but if you're resorting to personal attacks to make your point, you're doing it wrong, and you should feel bad.
```
remove comma before `and you` ❓
`doing it wrong and you should`

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
+2.  **Keep your posts and replies suitable for people of all-ages.** NSFW content isn't appropriate for an all-ages community, though there's nothing stopping you from discussing these sorts of things in private messages.
+3.  **Be productive with your criticism without resorting to personal attacks.** Criticism is a wonderful thing when done properly, but if you're resorting to personal attacks to make your point, you're doing it wrong, and you should feel bad.
+4.  **Keep threads relevant, and avoid resurrecting older topics if the conversation has died out.** If a topic has been dead for years, it isn't really relevant anymore, so just let it fade away. Post a new one, or maybe search and see if someone has answered your question already.
+5.  **Keep your signatures and avatars reasonable.** If they're animated, they shouldn't strobe or flash excessively, and they should be of a filesize small enough for users with slower connections to not feel their presence on the page.
```
comma splice?
`flash excessively. They should be of a`

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
+3.  **Be productive with your criticism without resorting to personal attacks.** Criticism is a wonderful thing when done properly, but if you're resorting to personal attacks to make your point, you're doing it wrong, and you should feel bad.
+4.  **Keep threads relevant, and avoid resurrecting older topics if the conversation has died out.** If a topic has been dead for years, it isn't really relevant anymore, so just let it fade away. Post a new one, or maybe search and see if someone has answered your question already.
+5.  **Keep your signatures and avatars reasonable.** If they're animated, they shouldn't strobe or flash excessively, and they should be of a filesize small enough for users with slower connections to not feel their presence on the page.
+6.  **Don't post advertisements, referral links or any other annoying or intrusive content.** Things like affiliate and referral links are fun for the person posting them and an annoyance for everybody else, so don't do it.  Autoloading or autoplaying content/embeds falls within this rule as well.
```
double space
```
so don't do it.  Autoloading
               ^ here
```

`so don't do it. Autoloading`

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
+2.  **Stay on topic in Beatmap discussions.** These threads/discussions are about the beatmap in question, and nothing else. If you have an issue with something not directly pertaining to the beatmap, post it in the appropriate forum.
+3.  **Do not plagiarise or attempt to steal the work of others.** Do not also upload or use other people's work without their explicit permission (including, but not limited to skins and guest difficulties).
+
+_See also: [Beatmap Submission Rules](/wiki/BSS)_
```
iirc, it is a convention to put the see also right after the section starts and not after it ends 😖 ....
also, BSS links to the glossary that just explains what BSS stands for... did you mean to link to Submission _(but even that doesn't make sense too)_?

Ephemeralis replied:
i'll just drop the BSS link for now and it can be added in later, it's a holdover from the last version of this ruleset

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
+These bad things may include (*but are not limited to*):
 
-Cheaters are permanently banned and restricted upon the first offense and are forced to wait 6 months MINIMUM before they may appeal the ban. Just don't do it. Bans are rarely, if ever, made in error.
+* The removal of your ability to chat in-game for a period of time (known as a silence)
```
full stop?

Ephemeralis replied:
oxford standards for bullet points do not include full stops at the end of each point

MegaApplePi replied:
gah, I have been doing that in a few articles tho iirc ¯\_(ツ)_/¯

Ephemeralis replied:
it's not a huge deal so long as it's consistent in the singular articles themselves

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
 
-Cheaters are permanently banned and restricted upon the first offense and are forced to wait 6 months MINIMUM before they may appeal the ban. Just don't do it. Bans are rarely, if ever, made in error.
+* The removal of your ability to chat in-game for a period of time (known as a silence)
+* The removal of any infringing content from your userpage, profile or uploaded beatmaps
```
full stop?

Ephemeralis replied:
see above

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
-Cheaters are permanently banned and restricted upon the first offense and are forced to wait 6 months MINIMUM before they may appeal the ban. Just don't do it. Bans are rarely, if ever, made in error.
+* The removal of your ability to chat in-game for a period of time (known as a silence)
+* The removal of any infringing content from your userpage, profile or uploaded beatmaps
+* The loss of the privilege to edit or alter your userpage or profile details
```
full stop?

Ephemeralis replied:
above redux

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
+* The removal of your ability to chat in-game for a period of time (known as a silence)
+* The removal of any infringing content from your userpage, profile or uploaded beatmaps
+* The loss of the privilege to edit or alter your userpage or profile details
+* The loss of the privilege to play or qualify in officially hosted tournaments
```
full stop?

Ephemeralis replied:
above redux: electric boogaloo

Ephemeralis commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
-3.  Do not steal others' beatmaps, presenting them as if they're mapped by you. Do not also upload or use other people's work without their explicit permission (including, but not limited to skins and guest difficulties).
+A set of more detailed rules for beatmap construction exists, known as the [Ranking Criteria](/wiki/Ranking_Criteria).
+
+1. **Make sure you have permission to use any content involved in your beatmap.** This includes songs, videos, hitsounds, graphics, and any other content that isn't your own creation. If you are unsure of where to find content that is free to use, we have a growing library of tracks available at the [Featured Artist](https://osu.ppy.sh/beatmaps/artists/) listing, all 100% licensed for use in osu! ...
```
the ! in osu! is part of the identifying semantic object and thus does not fill its role as an exclamation point, so a full-stop is grammatically accurate here.

Ephemeralis commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,81 @@
+Restricted accounts can not play online, set scores, chat with other users and are for all intents and purposes 'banned' from interaction with the community. Restriction for community rule violation can only be removed upon appeal to the game staff after 3 months have passed.
 
-Restriction is SERIOUS, and can only be revoked by a member of the player support staff, accessible via accounts@ppy.sh or the live chat support periodically available on all pages of the website.
+Severe offenses that go well beyond the scope of our rules and into the jursidiction of illegal content as per international common law will result in immediate account termination. This applies nearly exclusively to extremely graphic or otherwise abhorrent offenses, such as the distribution of child pornography or other material of abuse.
```
Good catch!

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
+6.  **Be mindful of what you share with others, especially if the content is potentially disturbing or very graphic in nature.** If you have to think twice about whether something is appropriate to link, it probably isn't okay. Avoid things with fast, strobing lights, gore, images of abuse or other graphic material.
+7.  **Keep links to relevant and legal websites.** Bootleg streaming websites (officially licensed ones like Animelab and Crunchyroll are OK) and anything related to "warez" or pirated software of any kind aren't okay, and we don't want them in our chat. Advertising Twitch links for other video games is okay, but ONLY in `#videogames`.
+8. **Don't advertise third-party chat servers of any kind, including Discord, Teamspeak or Skype servers/groups.** The osu! chat is about osu! foremost and not about sharing links to your personal servers. Do it via PM if you absolutely have to.
+9.  **In language-specific channels, you must speak the dominant language (English for #english, Thai for #thai, etc) if you are participating in discussion.** Where the language for a channel is not explicitly stated (or inferred from the channel name, as in Thai for `#thai` and Dutch for `#dutch`), it is always English. The `#lobby` and `#help` channels can be used with any and every language,  ...
```
uh, what happened? markdown derp
replace last sentence with this:
`*Members of the [moderation staff](/wiki/moderation_staff) do not need to follow this rule when doing their work.*`

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
+
+1. **Make sure you have permission to use any content involved in your beatmap.** This includes songs, videos, hit sounds, graphics, and any other content that isn't your own creation. If you are unsure of where to find content that is free to use, we have a growing library of tracks available at the [Featured Artist](https://osu.ppy.sh/beatmaps/artists/) listing, all 100% licensed for use in _os ...
+2.  **Stay on topic in Beatmap discussions.** These threads/discussions are about the beatmap in question, and nothing else. If you have an issue with something not directly pertaining to the beatmap, post it in the appropriate forum.
+3.  **Do not plagiarise or attempt to steal the work of others.** Do not also upload or use other people's work without their explicit permission (including, but not limited to skins and guest difficulties).
```
add a comma after `limited to`?
`including, but not limited to, skins and guest difficulties`

MegaApplePi commented on wiki/Rules/en.md:
```diff
@@ -1,77 +1,79 @@
+
+1. **Make sure you have permission to use any content involved in your beatmap.** This includes songs, videos, hit sounds, graphics, and any other content that isn't your own creation. If you are unsure of where to find content that is free to use, we have a growing library of tracks available at the [Featured Artist](https://osu.ppy.sh/beatmaps/artists/) listing, all 100% licensed for use in _os ...
+2.  **Stay on topic in Beatmap discussions.** These threads/discussions are about the beatmap in question, and nothing else. If you have an issue with something not directly pertaining to the beatmap, post it in the appropriate forum.
+3.  **Do not plagiarise or attempt to steal the work of others.** Do not also upload or use other people's work without their explicit permission (including , but not limited to, skins and guest difficulties).
```
remove space between:
```
including , but
         ^ here
```

Ephemeralis replied:
Sublime's ripping my dick off with these extra whitespaces, gosh.

Ephemeralis commented:
I think we should be good, hopefully.
